policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/dependent.yaml

```
@@ -0,0 +1,15 @@
+description: Indiana provides this base exemption amount for each dependent.  
+values:
+  2021-01-01: 1_000
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: Indiana dependent exemptions amount 
+  reference:
+    - title: IC 6-3-1-3.5 "Adjusted gross income" (a)(3)-(4)
```

(3) is the filer amount not dependent.
```suggestion
    - title: IC 6-3-1-3.5 "Adjusted gross income" (a)(4)
```